README.md

```diff
@@ -1,250 +1,326 @@
-        backbone/
-           images..
-        default/
-           images..
-        ...
-        OpenWrt-ImageBuilder-....tar.xz
-        OpenWrt-SDK-....tar.xz
-        initrd/
-           images..
-        packages/
-           packages/<ARCH>
-              base/*.ipk
-              luci/*.ipk
-              packages/*.ipk
-              packages_berlin/*.ipk
-              routing/*.ipk
-           targets/MAINTARGET/SUBTARGET/packages/
-              *.ipk
-```
 
-As you notice there are several different image variants ("backbone", "default", etc.).
-These different *packages lists* are defined in `packages/`.
-See the "Features" section above for a description of the purpose of each package list.
-With the "OpenWrt-Imagebuilder" you can assemble your own image variant with your
-*packages lists* without having to compile everything yourself. The "OpenWrt-SDK" is
-the fastest way to build your own packages or programs without compiling OpenWrt itself.
-The "initrd" directory contains some initrd-images for netboot, which are required on
-some boards to initially install OpenWrt.
+As you notice there are several different image variants ("backbone", "default",
+etc.). 
++ These different *packages lists* are defined in `packages/`. See the
+    [wiki](https://wiki.freifunk.net/Berlin:Firmware#Image_Types_.28English.29)
+    for a description of the purpose of each package list.
++ With the "OpenWrt-Imagebuilder" you can assemble your own image variant with
+    your *packages lists* without having to compile everything yourself. The
+    *OpenWrt-SDK* is the fastest way to build your own packages or programs 
+    without compiling OpenWrt itself.
++ The *initrd* directory contains some initrd-images for netboot, which is
+    required on some boards to initially install OpenWrt.
+
 
 ### customizing make
 
 `make` will use by default `TARGET` and `PACKAGES_LIST_DEFAULT` defined in
 `config.mk`. You can customize this by overriding them:
 
-```
-make TARGET=mpc85xx PACKAGES_LIST_DEFAULT=backbone
-```
-in addition you can build your own image from a prebuilt imagebuilder by something like:
+``` make TARGET=mpc85xx PACKAGES_LIST_DEFAULT=backbone ```
 
-```
-make images IB_FILE=<file> TARGET=... PACKAGES_LIST_DEFAULT=...
-```
+in addition you can build your own image from a prebuilt imagebuilder by
+something like:
 
-The default target is `ar71xx-generic`. For a complete list of supported targets look in `configs/` for the target-specific configs.
-Each of these targets need a matching file in `profiles/` with the profiles (boards) that should be build with the imagebuilder.
+``` make images IB_FILE=<file> TARGET=... PACKAGES_LIST_DEFAULT=... ```
 
-additional options
+The default target is `ar71xx-generic`. For a complete list of supported targets
+look at `configs/` for the target-specific configs. Each of these targets need a
+matching file in `profiles/` with the profiles (boards) that should be build
+with the imagebuilder.
```

Comment:
"should be build **by** the imagebuilder"?

Reply:
"sould be built by" would be the queen's english.